book/chapters/chapter1/introduction_and_overview.qmd

```diff
@@ -116,7 +116,7 @@ Throughout this book we will use the following code style:
 4. When referencing functions, we will only include the package prefix (e.g., `pkg::function`) for functions outside the `mlr3` universe or when there may be ambiguity about in which package the function lives. Note you can use `environment(function)` to see which namespace a function is loaded from.
 5. We denote packages, fields, methods, and functions as follows:
 
-    * `package` - With link (if online) to package CRAN, R-Universe, or GitHub page
+    * `package` with link (if online) to package CRAN, R-Universe, or GitHub page
```

Comment:
```suggestion
    * `package`
```

I think we should just remove that bit as we don't always link (only really first time)